installer/manifests/keptn/charts/control-plane/templates/approval-service.yaml

```diff
@@ -65,21 +65,36 @@ spec:
               value: 'http://localhost:8081/event'
             - name: LOG_LEVEL
               value: {{ .Values.logLevel | default "info" }}
-          {{- include "control-plane.common.container-security-context" . | nindent 10 }}
-        - name: distributor
-          image: {{ .Values.distributor.image.repository }}:{{ .Values.distributor.image.tag | default .Chart.AppVersion }}
-          {{- include "control-plane.dist.livenessProbe" . | nindent 10 }}
-          {{- include "control-plane.dist.readinessProbe" . | nindent 10 }}
-          imagePullPolicy: IfNotPresent
-          ports:
-            - containerPort: 8080
-          {{- include "keptn.distributor.resources" . | nindent 10 }}
-          env:
-            - name: PUBSUB_TOPIC
-              value: 'sh.keptn.event.approval.>'
-            - name: PUBSUB_RECIPIENT
-              value: '127.0.0.1'
-          {{- include "control-plane.dist.common.env.vars" . | nindent 12 }}
+            - name: K8S_DEPLOYMENT_NAME
```

Comment:
can we use what we previously had in `{{- include "control-plane.dist.common.env.vars" . | nindent 12 }}` here as well? Otherwise we will have a lot of duplicates when other services are updated as well

Reply:
resolved